### ISSUE TITLE
Recurring Help Text not appearing when 'I want to contribute this amount every month' checked

### ISSUE THREAD

**Issue by cmmadmin** [iATSPayments/com.iatspayments.civicrm#245]
When 'I want to contribute this amount every month' is checked the Recurring Help Text does not appear below it as it should.

Using the debugger I see that $paymentObject is NULL on line 546 of sites\all\modules\civicrm\CRM\Contribute\Form\Contribution\Main.php  which causes the recurringHelpText to be skipped.

Sorry for the influx of issues. Is this related to the other issue I posted perhaps?

CiviCRM 5.0.0, iATS 1.6.1

**Comment by KarinG** [comment]
Yes I believe it is related to the getCurrency regression/bug in Core. It’s on my list of things to look into; 

And no worries about posting to our GitHub queue 😀

**Comment by cmmadmin** [comment]
As a temporary fix, I am setting the recurringHelpText in a buildForm hook as follows. Please let me know when this is fixed so I can remove this hack. Thanks!

```
/** Implementation of hook_civicrm_buildForm */
function cmm_customization_civicrm_buildForm( $formName, &$form ) {
    if ( $formName == "CRM_Contribute_Form_Contribution_Main" ) {
      $form->assign('recurringHelpText', "Your recurring contribution will be processed automatically on the day of month of the initial transaction.");
    }
}
```
